requirements.txt

```diff
@@ -1 +1,24 @@
 
+ast-comments==1.0.1
+black==22.6.0
+coverage[toml]==7.3.2
+doc8==1.1.1
+flake8-isort==6.0.0
+flake8-pyproject==1.2.3
+flake8==6.0.0
+ghapi==0.1.19
+mypy==1.1.1
+pydantic==1.10.7
+pydocstyle==6.3.0
+pydoctest==0.1.22
+pylint==3.2.7
+pyspelling==2.7.3
+pytest==6.2.5
+regex==2023.3.23
```

Comment:
revert changes, these requirements are written in requirements_qa.txt

Reply:
these changes were done by Artyom Mikhailovich, without this file mypy didn`t work

Reply:
I can state that you can delete everything in this file and mypy will work correctly

Reply:
can you answer me for what reason you have this file and why are you filled it?

Reply:
I`m so sorry, I had the problem with pytests. To solve this issue, Artyom Mikhailovich fixed it